README.md

```diff
@@ -1,2 +1,183 @@
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Add a link to your deployed project.
+
+- [Live Demo Link](https://ilyassatlassi.github.io/TODO-list2/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+Creating your "Awesome book" project
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+<ul>
+-A Git hub account
+-Git bash
+-Node JS
+-Visual Studio Code as your code editor
+</ul> 
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git clone[ (https://github.com/ilyassatlassi/TODO-list2.git)]
+```
+### Install
+
+### Usage
+
+### Run Tests
+
+Clone this repository to your desired folder:
+
+```sh
+  Open in VS code or any browser
+```
+
+### DEPLOY
+
+Clone this repository to your desired folder:
+
+```sh
+  Open with git pages.
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+ 
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
+
+👤 **Author1**
+
+- GitHub: [@ilyassatlassi](https://github.com/ilyassatlassi)
+- Twitter: [@ilyassatlassi](https://twitter.com/ilyass_atlassi)
+- LinkedIn: [@ilyassatlassi](https://www.linkedin.com/in/www.linkedin.com/in/ilyassatlassi)
+
+👤 **Author2**
+
+- GitHub: [@soliman](https://github.com/adelazizsoliman)
+- Twitter: [@soliman](https://twitter.com/Abdelaz21958317)
+- LinkedIn: [soliman](https://www.linkedin.com/in/abdelaziz-soliman-16989b89/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Future Features <a name="Future-features"></a>
+
+> features you will add to the project.
+
+- **[Adding_styles]**
+- **[Adding_navigation_links]**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+ 
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+> If you like this project, you can contribute to the project.
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
+
+
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
+
```

Comment:
- [ ] I suggest that you remove this since it doesn't have any importance to users or visitors. It's part of the ReadMe template that's at guiding you.